tensor.go

```diff
@@ -116,6 +148,26 @@ func MM(a, b Tensor) Tensor {
 	return Tensor{&t}
 }
 
+// LeakyRelu returns leaky relu of the tensor according to negativeSlope
+func LeakyRelu(t Tensor, negativeSlope float64) Tensor {
+	return t.LeakyRelu(negativeSlope)
+}
+
+// Relu returns relu of the tensor
+func Relu(t Tensor) Tensor {
```

Comment:
It seems that we provide two kinds of API:

First way:

```
a = RandN((100, 10)
b = a.Relu()
```

Second way:

```
a = RandN((100, 10)
b = Relu(a)
```

Which API do we prefer to use?

Reply:
The C++ and Python frontend also support these two styles of APIs. We'd better support both.

Reply:
Agree with the second way, it's much more like Python API
``` python
def forward(self, x):
    x = self.conv1(x)
    x = F.relu(x)
    ....
```

And some other function API maybe have more then one Tensor, e.g. 
``` python
torch.stack(tensors, dim=0, out=None) → Tensor
```
https://pytorch.org/docs/stable/generated/torch.stack.html#torch.stack

Reply:
It seems that `numpy` also support two kinds of APIs:

```python
import numpy as np
a = np.array([1.0, 2.0])
print(a.sum())
print(np.sum(a))
```

So, I feed well that we provide both.